wcvendors/wcvendors
# Show the Vendor selectbox while using Gutenberg

### All Submissions:






### Changes proposed in this Pull Request:



Closes #852.

### How to test the changes in this Pull Request:

1. Enable Gutenberg for Woocommerce Products
2. Try editing product
3. See vendor dropdown

### Other information:
